### ISSUE TITLE
Compile Error

### ISSUE THREAD

**Issue by src893** [AshamaneProject/AshamaneCore#260]
**Description:**

Compilation in error

**Current behaviour:**

"src/server/game/Entities/Player/Player.h:116:16: error: forward declaration of struct cannot have a nested name specifier struct RatedBattlefieldInfo::BracketInfo;"

**TC rev. hash/commit:** 

lartest

**Database version**

ADB latest

**Operating system:** CHANGEME OS

debian 10 Buster

**Comment by MttAI-dev** [comment]
Yes thee is compiling error any way now other with asio and still getting error fix please?

**Comment by src893** [comment]
> Yes thee is compiling error any way now other with asio and still getting error fix please?

what????

**Comment by ReyDonovan** [comment]
use flag warning 0 and not problem

**Comment by tach-tm** [comment]
"boost::asio::basic_waitable_timer<std::chrono::steady_clock,boost::asio::wait_traits<Clock>,boost::asio::any_io_executor>::basic_waitable_timer(const boost::asio::any_io_executor &,const std::chrono::time_point<std::chrono::steady_clock,std::chrono::nanoseconds> &)": cannot convert argument 1 from "boost::asio::executor" в "const boost::asio::any_io_executor &"	worldserver	worldserver\TCRest\server_http.h	296	

here is such an error to come out when compiling
